carma-messenger-core/cpp_message/src/MessageConsumer.cpp

```diff
@@ -60,6 +63,8 @@ namespace Message_cpp
     
     int MessageConsumer::run(){
         initialize();
+        cpp_message::Message geofence_node;
+        geofence_node.run();
```

Comment:
I'm not 100% sure this works as expected, I'm concerned the current way still only gets one (geofence vs other messages) portion of this code running at a time. Putting `initialize()` before calling geofence node may help, but it still seems kind of wonky. Let's discuss a bit more on this if you have time.